app/src/main/java/net/tensory/rxjavatalk/models/DebtReport.java

```diff
@@ -0,0 +1,23 @@
+package net.tensory.rxjavatalk.models;
+
+public class DebtReport {
+    private Value<Double> v1;
```

Comment:
Could we rename these to something that refers to the house?

Reply:
I'm going to delete this class.